repos/system_upgrade/el7toel8/actors/checkhacluster/tests/test_check_ha_cluster_checkhacluster.py

```diff
@@ -16,7 +16,9 @@ def test_no_inhibit_when_no_ha_cluster(monkeypatch, current_actor_context):
 
 
 def test_inhibits_when_cluster_node(monkeypatch, current_actor_context):
-    monkeypatch.setattr("os.path.isfile", lambda path: True)
+    # NOTE(ivasilev) Limiting here the paths to mock not to cause unexpected side-effects
+    # (original test had path: True)
+    monkeypatch.setattr("os.path.isfile", lambda path: path in [COROSYNC_CONF_LOCATION, CIB_LOCATION])
```

Comment:
nit, I usually prefer tuples for constants like this. Lists are more expensive. No need to change, I just wanted to point this out.